ipaserver/install/replication.py

```diff
@@ -584,7 +584,7 @@ def setup_changelog(self, conn):
         )
         try:
             conn.add_entry(entry)
-        except errors.DuplicateEntry:
+        except (errors.DuplicateEntry, errors.DatabaseError):
```

Comment:
What would cause a DataBaseError? This would ignore the error and try to continue.

Reply:
Previously adding `cn=changelog5,cn=config` would return err=68 if it already exist, but with the change in https://pagure.io/389-ds-base/pull-request/51181#_15__191 it will return err=53, which is `errors.DatabaseError`, because separate changelog is no longer supported.
See also discussion in https://pagure.io/389-ds-base/pull-request/51101#comment-122648